.github/workflows/integration-tests.yml

```diff
@@ -4,41 +4,47 @@ on:
   workflow_dispatch:
   pull_request:
     branches:
+      - main
       - triton-mlir
 
 jobs:
 
   Integration-Tests:
 
-    runs-on: [self-hosted, A10]
-
+    strategy:
+      matrix:
+        os: [self-hosted, A10, ubuntu-latest, macos-latest]
```

Comment:
I don't think that works. [self-hosted, A10] need to be in the same tag group, as we want to select specifically self-hosted agents with the A10 tag

Reply:
Oh I see. I will update it.

Reply:
Updated. I confirmed that the current solution picks only `self-hosted` with the `A10` tag locally (https://github.com/shintaro-iwasaki/triton/pull/3).